src/DataCollection.UWP/MainPage.xaml.cs

```diff
@@ -73,5 +82,64 @@ private void BladeItem_Expanded(object sender, System.EventArgs e)
             var bladeItem = sender as BladeItem;
             bladeItem.BorderThickness = new Windows.UI.Xaml.Thickness(1);
         }
+
+        /// <summary>
+        /// Event handler for displaying a message to the user
+        /// </summary>
+        private async void UserPrompt_MessageValueChanged(object sender, UserPromptMessageChangedEventArgs e)
+        {
+            var contentDialog = new ContentDialog()
```

Comment:
I really like this UI for the confirmation dialogs, it feels very natural for UWP :)